Userland/Utilities/jail-attach.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    int jail_index = 0;
```

Comment:
the jail indices are u64 in the kernel, so if you make this one as well, that's one less error check below.